### PR TITLE
Add Note POS - Pay Someone Nearby

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
Added description to POS and Pay Someone Nearby Invoices
Merged addStandardInvoice-addInvoice into one.

### CODE REVIEW

File: lib/bloc/invoice/invoice_bloc.dart [hunk 1]
```diff
@@ -36,6 +33,7 @@ class InvoiceBloc {
   Stream<bool> get paidInvoicesStream => _paidInvoicesController.stream;
 
   Int64 payBlankAmount = Int64(-1);
+  String description = "";
```
Comment (roeierez) [inline lib/bloc/invoice/invoice_bloc.dart]: What is the purpose of this "description" field? I also don't see it assigned anywhere.

File: lib/routes/user/pay_nearby/pay_nearby_page.dart [hunk 2]
```diff
@@ -58,6 +59,7 @@ class PayNearbyState extends State<_PayNearbyPage> {
                     if (_formKey.currentState.validate()) {
                       _formKey.currentState.save();
                       widget._invoiceBloc.payBlankAmount = _amountToSendSatoshi;
+                      widget._invoiceBloc.description = _descriptionController.text;
```
Comment (erdemyerebasmaz) [inline lib/routes/user/pay_nearby/pay_nearby_page.dart]: > What is the purpose of this "description" field? I also don't see it assigned anywhere.

It is assigned here.

Comment (roeierez) [reply lib/routes/user/pay_nearby/pay_nearby_page.dart]: It is wrong to assign a bloc variable from outside. Everything in the bloc should be private except for the stream interfaces.
I see also previously we had payBlankAmount behaves like this. Let's fix it.

**Comment by erdemyerebasmaz** [comment]
Use sinks to assign payBlankAmount and description.

**Pull request closed without merging**